CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contribution Guide
+
+## Welcome Section
+- **Introduction**: Welcome dear contributors to the Tegel Design system project. We are very thankful for your interest in contributing.
```

Comment:
```suggestion
- **Introduction**: Welcome, dear contributors, to the Tegel Design system. We are very thankful for your interest in contributing.
```

Im thinking that we should remove the word "project". It may suggest that it's a permanent, evolving entity rather than a one-time or finite effort?